config/src/lib.rs

```diff
@@ -552,6 +543,17 @@ fn set_contract_from_file(config: &mut RelayerConfig, file: Option<String>) -> R
     Ok(())
 }
 
+/// Parse the relayer's decryption key from a string
+pub fn parse_decryption_key(key_str: Option<String>) -> Result<DecryptionKey, String> {
+    if let Some(k) = key_str {
+        DecryptionKey::from_hex_str(&k)
+    } else {
+        // Must print here as logger is not yet setup
+        println!("{}\n", "WARN: No fee decryption key provided, generating one".yellow());
```

Comment:
I see why it's hard to ensure that logging is set up before we call this function, but on the note of using `colored` here - in Datadog, this will just produce wonky text encodings of the color being used. Not really an issue since we can always search for this log, but FYI